presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedTestBase.java

```diff
@@ -633,49 +634,92 @@ public void testStringFilters()
     }
 
     @Test
-    public void testReadWriteNDVs()
+    public void testReadWriteStats()
```

Comment:
Should we add test case to show the behavior for iceberg configured with hive but do not config `hive_statistics_merge_strategy`?

Reply:
Added at https://github.com/prestodb/presto/blob/c449f368e438d518e0df21048244b20cd15d8cba/presto-iceberg/src/test/java/com/facebook/presto/iceberg/hive/TestIcebergHiveStatistics.java#L284-L300